docs/thrust/api_docs/utility/pair.rst

```diff
@@ -9,5 +9,4 @@ Pair
    :glob:
    :maxdepth: 1
 
-   ${repo_docs_api_path}/*typedef_group__pair*
```

Comment:
I get this error unless I remove this line:

```
/.../cccl/docs/thrust/api_docs/utility/pair.rst:8: WARNING: toctree glob pattern '/api/*typedef_group__pair*' didn't match any documents
```

Reply:
I get the same error locally (Ubuntu 24.04), but the CI seems to be fine. It could be a version mismatch somewhere.

Reply:
Weird. I see that page being written in CI builds. I’ll revert this.